Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -1021,27 +1021,24 @@ - (void)_uploadItemForImageData:(NSData *)imageData imageName:(NSString *)imageN
     successBlock(adoptNS([[_WKImageFileUploadItem alloc] initWithFileURL:[NSURL fileURLWithPath:filePath isDirectory:NO]]).get());
 }
 
-- (void)_uploadItemForJPEGRepresentationOfImage:(UIImage *)image successBlock:(void (^)(_WKFileUploadItem *))successBlock failureBlock:(void (^)(void))failureBlock
+- (void)_uploadItemForPNGRepresentationOfImage:(UIImage *)image successBlock:(void (^)(_WKFileUploadItem *))successBlock failureBlock:(void (^)(void))failureBlock
```

Comment:
I don't think we should change this one. Doesn't this mean that the image data we submit in a form submission will be PNG instead of JPEG?

Reply:
Ah, I see. I'll revert that bit.